modules/openapi-generator/src/main/resources/JavaSpring/libraries/spring-boot/openapi2SpringBoot.mustache

```diff
@@ -63,13 +63,13 @@ public class OpenAPI2SpringBoot implements CommandLineRunner {
                         .allowedMethods("*")
                         .allowedHeaders("Content-Type");
             }*/
-{{^useSpringfox}}
+{{^springFoxDocumentationProvider}}
```

Comment:
Is this the right check? Or is the resource handler needed for specific non-Springfox providers?

Reply:
I just replaced the template variable name - not the logic

Reply:
Hmmm... I suspect this will cause issues with some configurations but we can find out later